src/Shell/Task/ModelTask.php

```diff
@@ -273,9 +267,15 @@ public function findBelongsTo($model, array $associations)
                         $tmpModelName = Inflector::camelize($found);
                     }
                 }
+                $foreignKeys = (array)TableRegistry::get($tmpModelName)->primaryKey();
```

Comment:
This should be fixed now (there is a lookup for the table and a continue/break a few lines up)